tests/integration-tests/utils.py

```diff
@@ -22,6 +22,8 @@
 from assertpy import assert_that
 from retrying import retry
 
+from pcluster.utils import InstanceTypeInfo
```

Comment:
we do not want to introduce any dependency between the integ tests package and the CLI code. They should be two standalone entities. Let's copy the part of the code we need.

Reply:
Done